### DISCUSSION TITLE
Add wind_speed sensor device class

### DISCUSSION THREAD

**Discussion by epenet** [home-assistant/architecture#817] [Entity Models] [answered]
## Proposal

Add a `sensor` `device_class` `wind_speed` for wind speed to home assistant core and frontend (+ documentation) with supported units:
- `ft/s`
- `km/h`
- `kn`
- `mph`
- `m/s`

This sensor device class would be useful for most integrations providing weather data sensors.

The unit of a `wind_speed` sensor shall be configurable by the user in the same way as temperature and pressure sensors etc.

**Comment by MartinHjelmare** [comment, marked as answer]
I'd continue here instead:
https://github.com/home-assistant/architecture/discussions/775
